taxcalc/tests/test_dropq.py

```diff
@@ -215,45 +215,44 @@ def test_dropq_diff_table(groupby, res_column, puf_1991_path):
 @pytest.mark.requires_pufcsv
 def test_with_pufcsv(puf_path):
     # specify usermods dictionary in code
-    start_year = 2016
-    reform_year = start_year + 1
-    reforms = dict()
-    reforms['_II_rt3'] = [0.33]
-    reforms['_PT_rt3'] = [0.33]
-    reforms['_II_rt4'] = [0.33]
-    reforms['_PT_rt4'] = [0.33]
+    start_year = 2017
+    reform_year = start_year
+    analysis_year = 2026
+    year_n = analysis_year - start_year
+    reform = {
+        '_FICA_ss_trt': [0.2]
+    }
```

Comment:
Just to confirm: the reason of switching to this reform is that it will change not only iitax but also payroll right?